doc/rst/developer/bestPractices/buildingdocs.rst

```diff
@@ -1,15 +1,230 @@
 .. _readme-buildingdocs:
 
+.. _Chapel Developers: https://chapel.discourse.group/c/developers
+
 =============================
 Building Chapel Documentation
 =============================
+The live Chapel web documentation lives at https://chapel-lang.org/docs/.
+All of the source for the documentation is in the chapel source repository, and
+this note talks about how those are organized and built into the html documentation
+seen on the website.
 
 The easiest way to build the Chapel web documentation is to use the top level
-make target::
+of your chapel github clone
 
+make target::
    [g]make docs
 
 This will build the documentation for the standard and internal modules using
 chpldoc, and symlink all of the ``$CHPL_HOME/doc/`` documents over to
 a sphinx project that lives in ``$CHPL_HOME/doc/``. Once built,
 the documentation can be viewed at ``$CHPL_HOME/doc/html/index.html``.
+
+The html documentation is also generated by the github CI (continuous integration)
+process.  You can go to your pull request in github, click on "Checks",
```

Comment:
As above, do we want GitHub to be capitalized or not?

Reply:
Yes.  Done.